backend/app/views.py

```diff
@@ -109,11 +110,11 @@ def get_sentiment_analysis(request):
     API endpoint for generating audio based on the sentiment analysis of the given text
     """
     text = request.query_params.get('text')
-    audio_metadata = text_to_sound(text)
+    audio_data = text_to_sound(text)
 
-    audio = audio_metadata['audio_samples']
-    sample_rate = audio_metadata['sample_rate']
-    encoded_audio = wav_to_base64(audio, sample_rate)
+    # audio_data = filters.change_pitch(audio_data, 3)
```

Comment:
Is this comment meant to demonstrate how one might implement one of your new filters?

Reply:
This was from us testing out the function by playing sound from the webpage. This can be deleted.

Reply:
It can be! We were using that line to test out some of the filters using the frontend. We were planning on removing the line (perhaps after the hack session), but we could leave it there for future reference!